src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -1246,6 +1253,28 @@ private static TypeSymbol RefineAccessThroughType(LookupOptions options, TypeSym
         }
 
         /// <summary>
+        /// A symbol is accessible for referencing in a cref if it is in the same assembly as the reference
+        /// or the symbols's effective visibility is not private.
+        /// </summary>
+        private bool IsCrefAccessible(Symbol symbol)
+        {
+            return !IsEffectivelyPrivate(symbol) || symbol.ContainingAssembly == this.Compilation.Assembly;
+        }
+
+        private static bool IsEffectivelyPrivate(Symbol symbol)
+        {
+            for (Symbol s = symbol; (object)s != null; s = s.ContainingSymbol)
+            {
+                if (s.DeclaredAccessibility == Accessibility.Private)
```

Comment:
Aren't there some symbols that don't have DeclaredAccessibility?  Are any of them effectively private?